jscomp/core/lam_compile_primitive.ml

```diff
@@ -36,8 +36,47 @@ let ensure_value_unit (st : Lam_compile_context.continuation) e : E.t =
   | EffectCall Not_tail -> e
 (* NeedValue should return a meaningful expression*)
 
-let translate loc (cxt : Lam_compile_context.t) (prim : Lam_primitive.t)
-    (args : J.expression list) : J.expression =
+let rec module_of_expression = function
+  | J.Var (J.Qualified (module_id, value)) -> [ (module_id, value) ]
+  | J.Caml_block (exprs, _, _, _) ->
```

Comment:
What set of cases is this match expected to cover?

Reply:
No need anymore. It was for the packaged module `module(Belt.List)`. Now the packaged module will be unpacked.